packages/amplify-cli/src/extensions/amplify-helpers/update-amplify-meta.ts

```diff
@@ -59,6 +61,21 @@ export const updateAwsMetaFile = (
   return amplifyMeta;
 };
 
+const copyFromVFStoFSSync = (src: string, dest: string): void => {
```

Comment:
Because this isn't intended to be a general purpose `cp` command, could we revise to accept a single path that we walk recursively?